themes/default/layouts/registry/api.html

```diff
@@ -31,7 +31,7 @@ <h1>{{ .Params.title }}: API Docs</h1>
             </div>
         </div>
     </div>
-    <div class="lg:w-3/12 lg:pl-8 min-w-max hidden xxl:block">
+    <div class="lg:pl-8 hidden xxl:block" style="max-width: 410px">
```

Comment:
I think you can still have the `lg:w-3/12` here, it will just respect the max width in the case where 3/12s of the screen is more than 410px

Reply:
good catch. yeah just added it back!